example/test_problem/ELBDM/LSS_Hybrid_Zoomin/plot_script/Compute_profiles.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3.9
+################################################################################
+
+# This code outputs the density, cumulative mass, circular velocity and velocity dispersion profiles
+# of a target halo. The code uses an initial estimate (center_first_guess) to search the vicinity within 
+# a specified radius (vicinity) for the coordinates of the maximum density,
+# corresponding to the center of the target halo.
+
+# carefully adjust vicinity for performance.
+
+# Example usage after running simulation with low resolution IC: 
+#    python3 Compute_profiles -s 36 -e 36
+
+# Outputs:
+#   Halo_Parameter
+#   prof_dens/Data_0000xx_0_profile_data
+#   prof_mass/Data_0000xx_0_mass_accumule
+#   prof_dens/Data_0000xx_0_profile_data
+#   prof_circular_vel/Data_0000xx_0_circular_velocity
+#   prof_veldisp/Data_0000xx_0_veldisp_haloRestFrame
+
+################################################################################
+import argparse
+import numpy as np
+import yt
+import sys
+from Profile_Functions import *
+
+# load the command-line parameters
+parser = argparse.ArgumentParser( description='Plot profile and output Halo_parameter' )
+
+parser.add_argument( '-s', action='store', required=True,  type=int, dest='idx_start',
+                      help='first data index' )
+parser.add_argument( '-e', action='store', required=True,  type=int, dest='idx_end',
+                      help='last data index' )
+parser.add_argument( '-d', action='store', required=False, type=int, dest='didx',
+                      help='delta data index [%(default)d]', default=1 )
+
+# take note
+print( '\nCommand-line arguments:' )
+print( '-------------------------------------------------------------------' )
+print( ' '.join(map(str, sys.argv)) )
+print( '-------------------------------------------------------------------\n' )
+
+args        = parser.parse_args()
+idx_start   = args.idx_start
+idx_end     = args.idx_end
+didx        = args.didx
+
+for file_id in range( idx_start, idx_end+1, 1 ):
```

Comment:
[Bug?] I think the for loop should be:
```suggestion
for file_id in range( idx_start, idx_end+1, didx ):
```